lib/Runtime/Base/FunctionBody.cpp

```diff
@@ -9611,6 +9611,15 @@ namespace Js
 #endif
                 if (validationCookie == currentCookie)
                 {
+                    if (this->jsMethod == reinterpret_cast<Js::JavascriptMethod>(this->GetNativeAddress()))
```

Comment:
>if (this->jsMethod == reinterpret_cast<Js::JavascriptMethod>(this->GetNativeAddress())) [](start = 20, length = 87)

When "jsMethod != ...", what would "jsMethod" contain, could it still create other false positives? Is it ok to always do this without checking "jsMethod != ..."?